TDDExercise/src/com/tdd/TaxCalculator.java

```diff
@@ -0,0 +1,54 @@
+package com.tdd;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+
+public class TaxCalculator {
```

Comment:
look at proposed changes in attachment, tests would have to be adjusted to account for corrected totals
[TaxCalculator.txt](https://github.com/novelwrites/tdd/files/4954166/TaxCalculator.txt)